src/copilot/utils/dataPlaneUtil.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import * as vscode from 'vscode';
+import { ApiCenterDataPlaneService, DataPlaneAccount } from "../../azure/ApiCenter/ApiCenterDataPlaneAPIs";
+import { AzureDataSessionProviderHelper, generateScopes } from "../../azure/azureLogin/dataSessionProvider";
+import { DataPlaneAccountsKey } from "../../constants";
+import { ext } from "../../extensionVariables";
+import { getSubscriptionContext } from "../../tree/DataPlaneAccount";
+import { UiStrings } from '../../uiStrings';
+import { GeneralUtils } from "../../utils/generalUtils";
+
+export async function createApiCenterDataPlaneService(): Promise<ApiCenterDataPlaneService> {
+    const account = await getOrSelectActiveAccount();
+    const scopes = generateScopes(account.clientId, account.tenantId);
+    const authSession = await AzureDataSessionProviderHelper.getSessionProvider().getAuthSession(scopes);
+
+    if (GeneralUtils.failed(authSession)) {
+        throw new Error(`Please sign in to Azure in 'API Center Portal View' Tree View. Error: ${authSession.error}`);
+    }
+
+    const subscriptionContext = getSubscriptionContext(account);
+    const apiCenterDataPlaneService = new ApiCenterDataPlaneService(subscriptionContext);
+
+    return apiCenterDataPlaneService;
+}
+
+export async function getOrSelectActiveAccount(): Promise<DataPlaneAccount> {
+    const accounts = ext.context.globalState.get<DataPlaneAccount[]>(DataPlaneAccountsKey, []);
+
+    if (accounts.length === 0) {
+        throw new Error("No Data Plane account found. Please trigger `Connect to an API Center` VS Code command to add Data Plane account");
```

Comment:
APIC Ext both support 'connect to an API Center' from command palette or deep link, it better to update this sentence as `please add Data plane account first, You can trigger command 'Connect to an API center' and sign in to connect this account` something like this.